docs/sample_config.yaml

```diff
@@ -278,6 +278,17 @@ listeners:
 # Used by phonehome stats to group together related servers.
 #server_context: context
 
+# Resource-constrained Homeserver Settings
+#
+# If limit_large_remote_room_joins is True, the room complexity will be
+# checked before a user joins a new remote room. If it is above
+# limit_large_remote_room_complexity, it will disallow joining or
+# instantly leave.
+#
+# Uncomment the below lines to enable:
+#limit_large_remote_room_joins: True
+#limit_large_remote_room_complexity: 1.0
```

Comment:
I think it'd be more consistent to have:

```yaml
limit_remote_rooms:
    enable: True
    complexity: 1.0
```

Reply:
yeah, sounds good